pr_check.sh

```diff
@@ -27,6 +27,15 @@ source $CICD_ROOT/deploy_ephemeral_env.sh
 echo "sleeping for 5 min"
 sleep 5m
 
+# Creating perf profile
+echo "Starting to create perf profile"
+service="kruize-recommendations"
+oc expose svc/${service} -n ${NAMESPACE}
+SERVER_IP=($(oc status --namespace=${NAMESPACE} | grep ${service} | grep port | cut -d " " -f1 | cut -d "/" -f3))
+echo "IP = $SERVER_IP"
+AUTOTUNE_URL="http://${SERVER_IP}"
+curl -s -H 'Accept: application/json' ${AUTOTUNE_URL} -d @./resource_optimization_openshift.json 
```

Comment:
Change the curl url to ${AUTOTUNE_URL}/createPerformanceProfile. You can change AUTOTUNE to KRUIZE